helmfile.d/15-ory.yaml

```diff
@@ -0,0 +1,85 @@
+bases:
+- ../environments.yaml
+
+---
+
+releases:
+  - name: kratos
+    chart: ory/kratos
+    version: {{ .Values.kratos._chart_version }}
+    installed: {{ .Values.kratos._install }}
+    timeout: {{ add .Values.base_timeout .Values.kratos._extra_timeout }}
```

Comment:
Please also add <<: *logFailedRelease to the release, similar to other ones.

Reply:
Done!